### ISSUE TITLE
FT does not pickup newly installed commands.

### ISSUE THREAD

**Issue by hrzafer** [felixse/FluentTerminal#231]
I've just installed heroku CLI for windows. When I created a new cmd tab (or a git-bash tab), `heroku` command was not recognized. When I started a new native cmd windows, the command was available. Restarting FT didn't work either. As I suspected, only after exiting from the system tray icon and starting it again worked. 

I think a new tab should be able to pick up changes in `PATH`. At least this behavior should be documented. Especially if user has selected  `show in system tray` in general settings, it should be documented that closing the FT windows has no effect.

**Comment by ericcornelissen** [comment]
> I think a new tab should be able to pick up changes in PATH

I think this is the only viable option, and this issue can be considered a bug :thinking:

**Comment by felixse** [comment]
This is pretty good explained here: https://github.com/felixse/FluentTerminal/issues/131
We would need to implement something similar to: https://stackoverflow.com/questions/171588/is-there-a-command-to-refresh-environment-variables-from-the-command-prompt-in-w

But yes, a bit of documentation on this wouldn't hurt

**Comment by felixse** [comment]
I'm extending the FAQ and will close this. It's already tracked in #131
